tests/yam/agama/auto.pm

```diff
@@ -4,19 +4,29 @@
 # Summary: First installation using D-Installer current CLI (only for development purpose)
 # Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
 
-use base 'y2_installbase';
+use base yam::agama::agama_base;
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
+use testapi;
 
+my $post_fail_pass;
 
 sub run {
+    $post_fail_pass = 'linux';
     assert_screen('agama-main-page', 120);
     assert_screen('agama-installing', 60);
 
     my @tags = ("welcome-to", "login");
     assert_screen \@tags, 960;
+    if (match_has_tag("login") || match_has_tag("welcome-to")) {
+        $post_fail_pass = 'nots3cr3t';
```

Comment:
ideally we should set it to undef (so it would use default) and not use LIVECD or LIVETEST variables.

Reply:
Why the initial case set LIVECD or LIVETEST ?

Reply:
I suppose it's because Agama runs on a livecd, but those variables were here for older livecds that were pretty different